<launch>
  <!-- Launch the grid map simple demo node -->

    <arg name="tf_frame" default="world" />
    <arg name="mesh_resolution" default="1.0" />
    <arg name="resolution" default="0.3" />
    <arg name="visualize_max_z" default="2.5" />
    <arg name="sensor_max_range" default="5.0" />
    <arg name="param_file" default="$(find rrt_explorer)/resource/exploration.yaml" />
    <arg name="debug" default="false"/>

<!--
<node pkg="victim_localization" type="test_NBV" name="test_NBV" output="screen" launch-prefix="gdb -ex run (double-)args" >
       <param name="tf_frame" type="string" value="$(arg tf_frame)" />
       <param name="resolution" type="double" value="$(arg resolution)" />
       <param name="visualize_max_z" type="double" value="$(arg visualize_max_z)" />
       <param name="sensor_max_range" type="double" value="$(arg sensor_max_range)" />
       <param name="mesh_resolution" type="double" value="$(arg mesh_resolution)" />
       <param name="map_publish_frequency" type="double" value="1.0" />
       <rosparam command="load" file="$(arg param_file)" />

</node>
-->
<node pkg="victim_localization" type="test_NBV" name="test_NBV" output="screen">
       <param name="tf_frame" type="string" value="$(arg tf_frame)" />
       <param name="resolution" type="double" value="$(arg resolution)" />
       <param name="visualize_max_z" type="double" value="$(arg visualize_max_z)" />
       <param name="sensor_max_range" type="double" value="$(arg sensor_max_range)" />
       <param name="mesh_resolution" type="double" value="$(arg mesh_resolution)" />
       <param name="map_publish_frequency" type="double" value="1.0" />
       <rosparam command="load" file="$(arg param_file)" />
       <rosparam file="$(find victim_localization)/config/costmap_params.yaml" command="load" ns="costmap" />
</node>



  <!-- Launch the grid map visualizer -->
  <node pkg="grid_map_visualization" type="grid_map_visualization" name="grid_map_visualization" output="screen">
    <rosparam command="load" file="$(find victim_localization)/config/victim_Map_DL.yaml" />
  </node>

  <!-- Launch costmap2d -->
  <!--<node name="costmap_node" pkg="costmap_2d" type="costmap_2d_node" >
          <rosparam file="$(find victim_localization)/config/costmap_params.yaml" command="load" />
  </node>
-->

<!-- Launch RViz with the demo configuration -->
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find victim_localization)/rviz/test_NBZ.rviz" /> 


 <!-- <node pkg="tf" type="static_transform_publisher" name="_to_map_link" args="0 0 0 0 0 0  /map /world 500"/>  --> 

</launch>
